stumpy/aamp.py

```diff
@@ -13,7 +13,7 @@
 @njit(
     # "(f8[:], f8[:], i8, b1[:], b1[:], f8, i8[:], i8, i8, i8, f8[:, :, :],"
     # "f8[:, :], f8[:, :], i8[:, :, :], i8[:, :], i8[:, :], b1)",
-    fastmath=True,
+    fastmath=config.STUMPY_FASTMATH,
```

Comment:
There is a small part of me that is preferring the more explicit `config.STUMPY_FASTMATH_TRUE` instead of simply `config.STUMPY_FASTMATH`. The former is unambiguous. I feel like we'll still need to look up if `config.STUMPY_FASTMATH` is the one with a dictionary or not. What do you think @NimaSarajpoor?

Reply:
I actually had the experience and needed to check it a couple of times before. So, a clearer name can be helpful here!

Reply:
Okay! Let's do `config.STUMPY_FASTMATH_TRUE` and `config.STUMPY_FASTMATH_FLAGS` then.